inc/command/class-command.php

```diff
@@ -487,6 +518,58 @@ protected function run_command( InputInterface $input, OutputInterface $output,
 		return $return_val;
 	}
 
+	/**
+	 * Bootstrap tests folder.
+	 *
+	 * @param string $tests_folder Target folder to boostrap the tests into.
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 *
+	 * @return void
+	 */
+	protected function bootstrap_codecept( string $tests_folder, InputInterface $input, OutputInterface $output ) {
+		$default_suites = $this->get_test_suites( 'vendor/altis/dev-tools/tests' );
+
+		$suites = $input->getArguments()['options'][1] ?? '';
+		if ( 0 === strpos( $suites, '-' ) ) {
+			$suites = $default_suites;
+		} else {
+			$suites = explode( ',', $suites );
+			$invalid = array_diff( $suites, $default_suites );
+			if ( count( $invalid ) ) {
+				throw new Exception(
+					sprintf(
+						'Invalid suites selected: "%s", available suites are: "%s".',
+						implode( ',', $invalid ),
+						implode( ',', $default_suites )
+					)
+				);
+			}
+		}
+
+		$base_path = getcwd(); // Do we have another way to detect this ?
+		$tests_folder = $base_path . '/' . $tests_folder;
+
+		if ( ! file_exists( $tests_folder ) ) {
+			mkdir( $tests_folder );
```

Comment:
Might need to specify the permissions and recursive arguments too, path might be nested:
```suggestion
			mkdir( $tests_folder, 0755, true );
```